temporal-testing-junit5/LICENSE.txt

```diff
@@ -0,0 +1,18 @@
+Temporal Java SDK
+
+Copyright (c) 2020 Temporal Technologies, Inc. All Rights Reserved
+
+Copyright (c) 2017 Uber Technologies, Inc. All Rights Reserved
+
+AWS Simple Workflow Flow Library
+Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+file except in compliance with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
```

Comment:
I'm not sure if it's ok to use the same license file?

Reply:
Yes, this is fine.